boards/nxp/rd_rw612_bga/MX25U51245GZ4I00_FCB.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021-2023 NXP
```

Comment:
Assuming copyright is set like so because this file is based on the one in the HAL?

Reply:
yeah basically only formatting changes